File: .pub-cache/hosted/pub.dartlang.org/google_sign_in-6.1.4/example/integration_test/google_sign_in_test.dart
// Copyright 2013 The Flutter Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

import 'package:flutter_test/flutter_test.dart';
import 'package:google_sign_in/google_sign_in.dart';
import 'package:integration_test/integration_test.dart';

void main() {
  IntegrationTestWidgetsFlutterBinding.ensureInitialized();

  testWidgets('Can initialize the plugin', (WidgetTester tester) async {
    final GoogleSignIn signIn = GoogleSignIn();
    expect(signIn, isNotNull);
  });
}

File: .pub-cache/hosted/pub.dartlang.org/google_sign_in_android-6.1.16/example/android/app/src/androidTest/java/io/flutter/plugins/googlesigninexample/GoogleSignInTest.java
// Copyright 2013 The Flutter Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

package io.flutter.plugins.googlesigninexample;

import static org.junit.Assert.assertTrue;

import androidx.test.core.app.ActivityScenario;
import io.flutter.plugins.googlesignin.GoogleSignInPlugin;
import org.junit.Test;

public class GoogleSignInTest {
  @Test
  public void googleSignInPluginIsAdded() {
    final ActivityScenario<GoogleSignInTestActivity> scenario =
        ActivityScenario.launch(GoogleSignInTestActivity.class);
    scenario.onActivity(
        activity -> {
          assertTrue(activity.engine.getPlugins().has(GoogleSignInPlugin.class));
        });
  }
}
